packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -20,14 +25,38 @@ export default function BottomSheetRangeCell( props ) {
 		step = 1,
 		minimumTrackTintColor = '#00669b',
 		maximumTrackTintColor = Platform.OS === 'ios' ? '#e9eff3' : '#909090',
-		thumbTintColor = Platform.OS === 'ios' ? '#fff' : '#00669b',
+		thumbTintColor = Platform.OS === 'android' && '#00669b',
 		...cellProps
 	} = props;
+	const [ accessible, onChangeAccessible ] = useState( true );
+
+	const onCellPress = () => {
+		onChangeAccessible( false );
+		if ( this.sliderRef ) {
+			const reactTag = findNodeHandle( this.sliderRef );
+			AccessibilityInfo.setAccessibilityFocus( reactTag );
+		}
+	};
+
+	const accessibilityLabel =
+	sprintf(
+		/* translators: accessibility text. Inform about current spacer height value. %2$s: Spacer height current value. */
+		_x( '%1$s. Current value is %2$s px', 'range cell' ),
```

Comment:
Similarly not everything we adjust with a Slider will be in pixels, so I suggest removing px. 

and 'range cell' might not be an ideal description for translators. I am thinking sth like 'Slider for picking a number inside a range' ?

Reply:
To be honest I was not sure what should be there, so I will accept your suggestion 👍